.github/workflows/python-test-with-style.yml

```diff
@@ -30,9 +30,8 @@ jobs:
         make -j2
         export CEED_DIR=$PWD
         pip install .
-        cd tests/python
-        python setup-qfunctions.py build
-        pytest test-*.py --ceed /cpu/self/ref/serial -vv
+        cd python/tests
```

Comment:
Rename seems fine to me, and is clear if we ever split Python bindings into a new repository. Can you please just confirm that the tests don't end up in a wheel?

Reply:
I checked and they don't show up, at least on my machine